### PR TITLE
Start systemd symlink service in the role

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds tasks to this Ansible role that allow it to start the user-mode `systemd` service that creates a symlink to the COOL file share.

## 💭 Motivation and context ##

We want to enable this service for the VNC user in all our COOL AMIs, not [just in cisagov/pca-gophish-composition-packer](https://github.com/cisagov/pca-gophish-composition-packer/blob/develop/src/vnc.yml#L40-L95).

## 🧪 Testing ##

All `pre-commit` hooks and `molecule` tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] _All_ future TODOs are captured in issues, which are referenced in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [ ] All new and existing tests pass.

### CODE REVIEW

**Comment by jsf9k** [comment]
The Kali-based `molecule` testing will fail until cisagov/skeleton-ansible-role#92 is merged into this repository.  I'll rebase this branch and re-run the GitHub Actions once that happens.

**Review by dav3r — APPROVED** [review]
👍 👍

File: molecule/default/prepare.yml [hunk 1]
```diff
@@ -31,3 +31,21 @@
 
 - name: Import python playbook
   ansible.builtin.import_playbook: python.yml
+
+# This is necessary to setup for installing this Ansible role with the
+# specification that the test user have a user-mode systemd service
+# that creates a symlink to the /share/directory
```
Comment (mcdonnnj) [inline molecule/default/prepare.yml]:
```suggestion
# that creates a symlink to the /share directory.
```

**Review by mcdonnnj — APPROVED** [review]
Strong work 💪💪💪. I have one comment tweak for your consideration.

**Review by mcdonnnj — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
Still excellent! 👍

**Pull request merged**